### PR TITLE
Update repo with changes from cookiecutter, use cruft

### PR DESCRIPTION
**Pull request by tlambert03**
cruft updates!
(a little manual editing was required... but it made it very easy to know where)

### CODE REVIEW

**Review by Carreau — APPROVED** [review]
+1 and would be nice to release to avoid warnings on main napari test suite.

**Review by ziyangczi — APPROVED** [review]
Thanks for updating the plugin talley! 

Now that we are marking the svg with plugin trove classifier, shall we remove it from the napari default dependency and let user decide if they want to include it? I hope there is nothing in napari critically depend on this..

**Comment by tlambert03** [comment]
> I hope there is nothing in napari critically depend on this..

I don't think we were including it by default for lack of the classifier... but rather because it was considered generally useful enough.  but @sofroniewn can weigh in on whether he thinks it's time to remove?

File: tox.ini [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+platform = 
+    macos: darwin
+    linux: linux
+    windows: win32
+passenv = 
+    CI
+    GITHUB_ACTIONS
+    DISPLAY XAUTHORITY
+    NUMPY_EXPERIMENTAL_ARRAY_FUNCTION
+    PYVISTA_OFF_SCREEN
```
Comment (ziyangczi) [inline tox.ini:27]: out of curiosity, what is this? I don't recall seeing this anywhere in the code change

Comment (tlambert03) [reply tox.ini:27]: the passenv setting in tox is a list of environment variables that tox will pass from the calling context to the test environment, and they "generally" correspond with the `env:` section in the github test.yml (like [here in napari](https://github.com/napari/napari/blob/102a7e8f845893c874d2b86f9371d41130100b89/.github/workflows/test_pull_requests.yml#L140-L145))... 

however, good eye! not all of these are necessary or functional here :)  
We can remove `NUMPY_EXPERIMENTAL_ARRAY_FUNCTION` (which comes from napari tests when we are testing the minimum requirements), `PYVISTA_OFF_SCREEN` is required to get the Windows OpenGL off-screen buffer to work for things like screenshot tests, and is associated with the `Install Windows OpenGL` phase in the github action.  But here, we need to add something like `PYVISTA_OFF_SCREEN: True` to the github workflow for this to do any good in the tox.ini

**Review by jni — APPROVED** [review]

**Comment by jni** [comment]
@tlambert03 do you mind updating the title to be more descriptive? ;)

**Comment by sofroniewn** [comment]
> but @sofroniewn can weigh in on whether he thinks it's time to remove?

We could drop from main requirements. I might still include it in `napari[all]` though, at least as a temporary thing.

We have some tests in napari that require it, which is maybe a bit odd. Those should probably get pulled back into this repo/ dropped as needed

**Comment by sofroniewn** [comment]
also @tlambert03 feel free to push a new release here if you are ready!!

**Pull request merged**